extensions/amp-accordion/1.0/accordion.js

```diff
@@ -39,22 +41,29 @@ const AccordionContext = Preact.createContext(
 /** @type {!Object<string, boolean>} */
 const EMPTY_EXPANDED_MAP = {};
 
+const EMPTY_REAL_ID_MAP = {};
+
 const generateSectionId = sequentialIdGenerator();
 const generateRandomId = randomIdGenerator(100000);
 
 /**
  * @param {!AccordionDef.Props} props
+ * @param ref
  * @return {PreactDef.Renderable}
  */
-export function Accordion({
-  as: Comp = 'section',
-  expandSingleSection = false,
-  animate = false,
-  children,
-  id,
-  ...rest
-}) {
+function AccordionWithRef(
+  {
+    as: Comp = 'section',
+    expandSingleSection = false,
+    animate = false,
+    children,
+    id,
+    ...rest
+  },
+  ref
+) {
   const [expandedMap, setExpandedMap] = useState(EMPTY_EXPANDED_MAP);
+  const realIdMap = useRef(EMPTY_REAL_ID_MAP);
```

Comment:
Per note below, I think this can be avoided entirely.

Reply:
Removed.  Per off-line conversation allows API to directly access any section.  We will trust the users not to use the API incorrectly.